H/HELICS/build_tarballs.jl

```diff
@@ -39,11 +39,18 @@ if [[ "${target}" == *-mingw* ]]; then
 fi
 """
 
-products = [
-    LibraryProduct("libhelicsSharedLib", :libhelicsSharedLib),
-]
+products = if HELICS_VERSION < v"3.0.0"
```

Comment:
I mentioned in slack that you could also do
```julia
products = [
    LibraryProduct(["libhelicsSharedLib", "libhelics"], :libhelicsSharedLib),
]
```
to
1. automatically match both file names
2. retain the same name for the variable as before

The decision of what to do is up to you, of course, I'm just suggesting how to simplify this

Reply:
Thanks. I guess you are suggesting this because using the same code will not break any wrappers, right? I’m maintaining the wrapper in this case so I think it’s not a problem for me.

Reply:
Correct